beforepush.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+if ! command -v shellcheck; then
+	echo "shellcheck not found on this machine. Please install from https://www.shellcheck.net/"
+	exit 1
+fi
+
+if ! command -v terraform; then
+	echo "terraform not found on this machine. Please install from https://www.terraform.io"
+	exit 1
+fi
+
+files="$(find . -regex '.*\.sh' -not -regex '.*halyard.*')"
+for file in $files; do
+	echo "checking $file"
+	if ! shellcheck "$file"; then
+		echo "shellcheck failed for $file"
+		exit 1
+	fi
+done
+
+#setting up environment for terraform operations
+export VAULT_ADDR="http://FakeVaultAddr:8200"
```

Comment:
I would suggest only setting these if they are not already set

Reply:
moved to docker container where they are guaranteed not set.